build/Dockerfile

```diff
@@ -1,24 +1,25 @@
+ARG binary_name=kfctl
 FROM registry.access.redhat.com/ubi8/ubi-minimal:latest AS build
 
-ENV OPERATOR=/usr/local/bin/kfctl \
+ENV OPERATOR=/usr/local/bin/${binary_name} \
     USER_UID=1001 \
     USER_NAME=kfctl \
```

Comment:
does USER_NAME need to change as well?

Reply:
The username does not really influence anything in the way how things are setup in the Dockerfile, IMO, so no need for that